deltas/tokenizers/tokenizer.py

```diff
@@ -25,6 +30,18 @@ def from_config(cls, config, name, section_key="tokenizers"):
             return Tokenizer.from_config(config, name, section_key)
 
 
+class TokenizerPipeline(Tokenizer):
+    def __init__(self, tokenizer, *token_segmenters):
+        self.tokenizer = tokenizer  # start of the pipeline
+        self.token_segmenter = token_segmenters
```

Comment:
Should probably be called "Token Processor" like the abstract base class.

Reply:
I adjusted the name to make it more clear.